quickwit/quickwit-search/src/root.rs

```diff
@@ -151,25 +151,40 @@ pub struct IndexMetasForLeafSearch {
 }
 
 pub(crate) type IndexesMetasForLeafSearch = HashMap<IndexUid, IndexMetasForLeafSearch>;
-type TimestampFieldOpt = Option<String>;
+
+#[derive(Debug)]
+struct RequestMetadata {
+    timestamp_field_opt: Option<String>,
+    query_ast_resolved: QueryAst,
+    indexes_meta_for_leaf_search: IndexesMetasForLeafSearch,
+    sort_fields_are_datetime_opt: Vec<Option<bool>>,
```

Comment:
Do we need Option here or can we just use bool?

Reply:
We don't need the option here indeed, it's only useful in the validate function.

Reply:
I just removed it @PSeitz